pyproject.toml

```
@@ -4,7 +4,7 @@ requires = [
     "versioneer[toml]",
     "wheel",
     "numpy~=1.25",
-    "Cython~=0.29.36",
+    "Cython>0.29",
```

I think this should be 
```suggestion
    "Cython=3",
```
?

---

Done.

---

Unfortunately, this didn't work. 

I tried multiple things, only `Cython>=3.0.0` worked.

---

Weird, I find TOML formatting to be a headache. If it works it works 🤷